infection_monkey/network/info.py

```diff
@@ -138,15 +139,14 @@ def get_ips_from_interfaces():
     res = []
     ifs = get_host_subnets()
     for net_interface in ifs:
-        address_str = unicode(net_interface['addr'])
-        netmask_str = unicode(net_interface['netmask'])
-        host_address = ipaddress.ip_address(address_str)
+        address_str = net_interface['addr']
+        netmask_str = net_interface['netmask']
         ip_interface = ipaddress.ip_interface(u"%s/%s" % (address_str, netmask_str))
         # limit subnet scans to class C only
         if ip_interface.network.num_addresses > 255:
-            ip_interface = ipaddress.ip_interface(u"%s/24" % address_str)
-        addrs = [str(addr) for addr in ip_interface.network.hosts() if addr != host_address]
-        res.extend(addrs)
+            res.append(CidrRange(cidr_range="%s/24" % (address_str, )))
```

Comment:
Any reason we want to keep the class C limit?

Reply:
Was there previously. Logic says more than that is too much.
Your call whether to remove the limit.

Reply:
I'd rather remove these limits soon, or make it configurable. But  that depends on what we think is realistic. @ofriz ?

Reply:
Fixed